storage/fanout.go

```diff
@@ -240,27 +241,44 @@ func (q *mergeQuerier) Select(params *SelectParams, matchers ...*labels.Matcher)
 func (q *mergeQuerier) SelectSorted(params *SelectParams, matchers ...*labels.Matcher) (SeriesSet, Warnings, error) {
 	seriesSets := make([]SeriesSet, 0, len(q.queriers))
 	var warnings Warnings
+
+	wg := sync.WaitGroup{}
+	wg.Add(len(q.queriers))
```

Comment:
You don't need the waitgrup any more, and the channel here doesn't even need to be buffered

Reply:
Thanks for your review , done.
Thank you for your help. I have less than 1 year experience in go language. My previous work experience is using Java

![image](https://user-images.githubusercontent.com/9583245/74001391-5f455700-49a6-11ea-989d-e0384439c44b.png)